### PR TITLE
Size reduction

### LINKED ISSUE #119: Investigate improving load times
Sitting on 51216b8 right now and the popup takes almost 2 seconds to open and the dashboard takes at least 3. 0.4.2 is a lot faster (even if it's still pretty slow), is the difference because of dev vs. prod or is it because of recent changes? In any case, how can we improve load times?

### PR DESCRIPTION
**Pull request by ahnlabb**
Fixes #119

### CODE REVIEW

**Comment by Powersource** [comment]
Is this ready for review?

**Comment by Powersource** [comment]
The popup file seems to be small enough now, 2 files to go.

The heart icon in the popup is broken though.

File: src/popup/popup.vue [hunk 1]
```diff
@@ -72,7 +74,11 @@ export default {
     },
   },
   created() {
-    this.refresh();
+    import(/* webpackChunkName: "db" */ '../lib/db')
```
Comment (Powersource) [inline src/popup/popup.vue]: .js?

**Comment by ErikBjare** [comment]
What was the size difference?

**Pull request merged**